php/WP_CLI/Iterators/Transform.php

```diff
@@ -15,6 +16,7 @@ public function add_transform( $fn ) {
 		$this->transformers[] = $fn;
 	}
 
+	#[ReturnTypeWillChange]
```

Comment:
```suggestion
	#[\ReturnTypeWillChange]
```

Should we do this, instead of defining the `use` above?

Reply:
I honestly don't know but we can try.